Sources/CameraControlARView/ARViewContainer.swift

```diff
@@ -5,27 +5,40 @@
 //  Created by Joseph Heck on 2/9/22.
 //
 
+#if os(iOS)
+import UIKit
+#elseif os(macOS)
 import Cocoa
+typealias UIViewRepresentable = NSViewRepresentable
+#endif
 import RealityKit
 import SwiftUI
 
 /// A SwiftUI representable view that wraps an underlying augmented reality view with camera controls instance.
 ///
 /// Create an ``CameraControlARView`` externally and hand it into the container so that you can interact with the
 /// view controls, or the underlying scene, from within SwiftUI.
-public struct ARViewContainer: NSViewRepresentable {
+public struct ARViewContainer: UIViewRepresentable {
```

Comment:
I'd rather name anything in the code a more generic term if possible, to avoid misconceptions by anyone reading this later (including me!) - Can we call this "PlatformViewRepresentable" and make the relevant type aliases above at line 12 and another for the UIKit variant?